lib/streamlit/elements/json.py

```diff
@@ -29,36 +28,18 @@
 
 from streamlit.proto.Json_pb2 import Json as JsonProto
 from streamlit.runtime.state import SessionStateProxy
-from streamlit.type_util import is_iterable
 from streamlit.user_info import UserInfoProxy
 
 
 if TYPE_CHECKING:
     from streamlit.delta_generator import DeltaGenerator
 
-Body: TypeAlias = Union[List[Any], Tuple[Any, ...], MutableMapping[Hashable, Any]]
```

Comment:
Don't need the `TypeAlias` import anymore, nor the `Tuple`, `Hashable` or `MutableMapping` ones either, I believe.

Reply:
Thank you for your help 🥇 I've introduced changes which you've talked about 👍 

> Some editors and linting tools lets you know whether you have unused imports. Might be worth considering including in CI

This is an awesome idea, we have this in plan to add this both to CI and `pre-commit` cc: @sfc-gh-kbregula

Reply:
Glad to hear you have it in your plans!